.github/workflows/ci.yml

```diff
@@ -37,14 +37,28 @@ jobs:
                   python-version: ${{ matrix.py }}
                   condarc-file: etc/.condarc
 
-            - name: Install conda deps
+            - name: Install conda deps a
               run: |
                 conda info
                 conda list
-                conda install -y mamba
-                mamba install -y --file etc/standalone_conda_requirements.txt
-                conda info
-                mamba list
+
+            - name: Install conda deps b1
+              run: |
+                conda install -y gitpython dust_extinction batoid batoid-rubin sncosmo palpy
+
+            - name: Install conda deps b2
+              run: |
+                conda install -y rubin-sim=1.1.1
+
+            - name: Install conda deps b3
+              run: |
+                conda install -y stackvana>=0.2023.32
+
+            # - name: Install conda deps c
+            #   run: |
+            #     conda install -y --file etc/standalone_conda_requirements.txt
+            #     conda info
+            #     conda list
```

Comment:
Why all this?  I think we don't want to have to edit this file whenever we add a new requirement.  I think if we add something to our conda_requirements file, this should automatically get the update.

Reply:
This is me trial-and-erroring to fix up the CI.  Still haven't got it, but I'll clean it up after I get to a solution.